### ISSUE TITLE
assert.isFinite() is not a function

### ISSUE THREAD

**Issue by falconvk** [cypress-io/cypress#5669]
### Current behavior:

`assert.isFinite(Number(val))`

results in
```
Error:       TypeError: assert.isFinite is not a function
```

### Desired behavior:

According to [documentation](https://docs.cypress.io/guides/references/assertions.html#TDD-Assertions) .isFinite() should be available like any other assertion.

### Steps to reproduce:

In any test try `assert.isFinite(Number('test'))`

### Versions

- Cypress 3.5.0

**Comment by jennifer-shehane** [comment]
This assertion was added post 4.0 of Chai as mentioned here: https://github.com/chaijs/chai/pull/779 

Since we're still using `chai` 3.5.0, this actually does not exist, so the documentation is wrong. This will be fixed as part of our 4.0 release https://github.com/cypress-io/cypress/issues/4270

**Comment by jennifer-shehane** [comment]
I've removed this from our docs here: https://github.com/cypress-io/cypress-documentation/commit/71da719f59684b749eab40a589fa7c6e0b9fcbdb

**Comment by jennifer-shehane** [comment]
The code for this is done in the upcoming 4.0 release, but this has yet to be released. We'll update this issue and reference the [changelog](https://on.cypress.io/changelog) when it's released.

```js
it('test', () => {
  assert.isFinite(Number(10))
});
```

## 3.8.3

<img width="437" alt="Screen Shot 2020-01-29 at 4 22 51 PM" src="https://user-images.githubusercontent.com/1271364/73346099-af8c2d00-42b3-11ea-85b8-7a28a6a98941.png">


## 4.0.0

<img width="453" alt="Screen Shot 2020-01-29 at 4 23 38 PM" src="https://user-images.githubusercontent.com/1271364/73346117-b9159500-42b3-11ea-9af8-5d875c7c159e.png">

**Comment by flotwig** [comment]
Released in [`4.0.0`](https://on.cypress.io/changelog#4-0-0).
